app/models/message.rb

```diff
@@ -6,6 +6,7 @@
 #  content             :text
 #  content_attributes  :json
 #  content_type        :integer          default("text"), not null
+#  created_by          :string
```

Comment:
I think it is better to use content_attributes here.

Reply:
Added new `jsonb` column `additional_attributes`